i3_resurrect/treeutils.py

```diff
@@ -90,10 +90,12 @@ def get_workspace_tree(workspace, numeric):
             for ws in container['nodes']:
                 # Select workspace and trigger name and num field
                 if numeric:
-                    if (workspace.isdigit()
-                            and 'num' in ws
-                            and ws['num'] == int(workspace)):
-                        return ws
+                    if workspace.isdigit():
+                        if 'num' in ws and ws['num'] == int(workspace):
+                            return ws
+                    else:
+                        util.eprint('Invalid workspace number.')
```

Comment:
Why move this error message here? To me this is UI (CLI) logic and a UI related error message so it should be in the command function (where it was before).

Reply:
ok